src/modules/lists/lists.service.ts

```diff
@@ -13,4 +14,12 @@ export class ListsService {
     const list = new this.listModel(data);
     return await list.save();
   }
+
+  async findByUserId(_id: string): Promise<List[]> {
+    if (isObjectId(_id)) {
+      return await this.listModel.find({user: {_id}}).exec();
```

Comment:
In here let's also filter by `public`, the controller should decide the value of this filter based on the user roles.